aspnetcore/tutorials/getting-started-with-swashbuckle.md

```diff
@@ -83,16 +83,32 @@ Add the Swagger generator to the services collection in the `Startup.ConfigureSe
 
 ::: moniker-end
 
-::: moniker range=">= aspnetcore-2.1"
+::: moniker range="= aspnetcore-2.1"
```

Comment:
I assume you want this to display for both 2.1 and 2.2. Consider the following suggestion all throughout the doc:

```suggestion
::: moniker range=">= aspnetcore-2.1 <= aspnetcore-2.2"
```

Reply:
Made the change where possible. Wasn't aware of this syntax. Thanks!